CocoaLumberjack.podspec

```diff
@@ -28,13 +28,13 @@ Pod::Spec.new do |s|
   s.default_subspecs = 'Core'
 
   s.subspec 'Core' do |ss|
-    ss.source_files         = 'Classes/CocoaLumberjack.h', 'Classes/DD*.{h,m}', 'Classes/Extensions/*.{h,m}', 'Classes/CLI/*.{h,m}'
-    ss.private_header_files = 'Classes/DD*Internal.{h}'
+    ss.source_files         = 'Classes/Include/**/*.{h}', 'Classes/DD*.{h,m}', 'Classes/Extensions/*.{h,m}', 'Classes/CLI/*.{h,m}'
```

Comment:
the Podspec no longer matches the folder structure

Reply:
Yes...